force_wfmanager/left_side_pane/workflow_settings.py

```diff
@@ -32,6 +36,78 @@ def get_text(self, object, trait, index):
         return get_bundle_name(self.item)
 
 
+class TreeEditorHandler(Handler):
+    def new_mco_handler(self, editor, object):
```

Comment:
`object` is passed as a keyword argument inside traitsui magic, so unfortunately we do not have control on this name